IpmiFeaturePkg/IpmiPowerRestorePolicy/IpmiPowerRestorePolicy.c

```diff
@@ -52,16 +56,15 @@ IpmiPowerRestorePolicyEntry (
   //
   // Get platform power restore policy setting
   //
-  Status = GetPowerRestorePolicy (&PlatformPowerRestorePolicy);
-
+  Status = GetPolicy (&gPlatformPowerRestorePolicyGuid, NULL, &PlatformPowerRestorePolicy, &PolicySize);
   if (EFI_ERROR (Status)) {
-    DEBUG ((DEBUG_ERROR, "[%a] - GetPowerRestorePolicy: %r\n", __FUNCTION__, Status));
+    DEBUG ((DEBUG_ERROR, "%a: Failed to get platform power restore policy! %r\n", __FUNCTION__, Status));
     return Status;

```

Comment:
Should this cover the case where platform did not produce the policy similar to the null library? Currently, null library will return no change and success, so that the rest of the flow will continue. The new change will make this start to fail.

Reply:
Thanks Kun for the suggestion! I have add default implementation of platform power restore policy.